### PR TITLE
widminbt is now correct bt widget

### PR DESCRIPTION
**Pull request by kamilkrz**
This is minor change only to ability to be replaced by another bt widget and vice versa

### CODE REVIEW

File: apps/widminbt/widget.js [hunk 1]
```diff
@@ -1,15 +1,13 @@
-(()=> {
-  WIDGETS.minbt={area:"tr",width:NRF.getSecurityStatus().connected?0:15,draw:function() {
-    if(this.width<15)return;
+WIDGETS["bluetooth"] = {area: "tr", width: NRF.getSecurityStatus().connected ? 0 : 15, draw: function () {
+    if (WIDGETS.bluetooth.width < 15) return;
```
Comment (bobrippling) [inline apps/widminbt/widget.js:2]: You could keep `this` here if you prefer:
```suggestion
    if (this.width < 15) return;
```

**Review by bobrippling — COMMENTED** [review]
thanks, looks great - i'll merge tomorrow to give you chance to change that suggestion, happy either way

File: apps/widminbt/widget.js (hunk 1 above)
Comment (kamilkrz) [reply apps/widminbt/widget.js:2]: I used this example to mimic same way widbthide is using.
In fact I found widbthide is exact opposite of widminbt. 
It seems nice to keep it simple for both so one way or another seems fit but changes needs to be done on both apps.

**Comment by bobrippling** [comment]
Thanks for the PR!

**Pull request merged**